course-tools-quickstart.md

```diff
@@ -0,0 +1,17 @@
+# Course tools Quickstart
```

Comment:
This assumes that the students work with a studentified (part of the [course management tools](https://github.com/lightbend-training/course-management-tools)) version of the materials. We need to adapt the `run` script on the _RPi's_ to support this. same thing for the `copy` command on the laptop too.
In the short term, it's probably easier to adapt these instructions to work with the master materials.

Reply:
@kikiya Been reflecting on working with a studentified a bit more. I think we should go for it. As said, it impact `run` and `copy`. It should be possible to have one version of each command that covers both cases (working with a master repo or the studentified repo). We may have to add a command option to cover handing the second (optional) argument to the `copy` command which specifies that the artefact should be copied to one node only (with node number equal to the number passed as second argument). 
Copy syntax on master would then be:

`./copy [-node=ddd] ex_nr`

whereas on a studentified repo it would be:

`./copy [-node=ddd]`